### PR TITLE
Copy updates to marketing job ads

### PR DESCRIPTION
**Pull request by katie-mai**
@megaslug @jesssummerfield

Made amendments to marketing manager titles - please check and approve.

### CODE REVIEW

File: app/messages/en.js [hunk 1]
```diff
@@ -662,11 +662,15 @@ export default {
     description: '',
   },
   jobs_partnership_marketing_manager: {
-    title: 'Partnership Marketing Manager',
+    title: 'Marketing Manager - Partnerships',
```
Comment (jesssummerfield) [inline app/messages/en.js]: @katie-mai `-` should be written as `&ndash;`

File: app/messages/en.js [hunk 2]
```diff
@@ -662,11 +662,15 @@ export default {
...
     description: '',
   },
   jobs_marketing_manager_accountants: {
-    title: 'Marketing Manager Accountants',
+    title: 'Marketing Manager - Accountants',
```
Comment (jesssummerfield) [inline app/messages/en.js]: @katie-mai `&ndash'` rather than `-`

File: app/messages/en.js (hunk 1 above)
Comment (katie-mai) [inline app/messages/en.js]: I tried it and come up on the website as "&ndash;"
screen shot 2017-01-06 at 18 26 57

File: app/messages/en.js [hunk 3]
```diff
@@ -662,11 +662,15 @@ export default {
...
+    description: '',
+  },
+  jobs_product_marketing_executive: {
```
Comment (megaslug) [inline app/messages/en.js]: There's `jobs_product_marketing_executive` below. Might want to keep just one of these as duplicates will fail the build.

**Pull request merged**